tests/StoringGroceries.tex

```diff
@@ -16,6 +16,7 @@ \subsection{Focus}
 		\item \textbf{Cupboard:} The cupboard has 5 shelves between 0.30m and 1.80m from the ground and contains several objects (See \ref{rule:scenario_objects}).
 		\begin{itemize}
 		 	\item \textbf{Door:} The cupboard has a single door, which is closed initially.
+		 	This door encloses some of the objects, covering up to one half of the cupboard (e.g. the left/right...), as indicated by the hatched area in the image.
```

Comment:
You took my proposal way too literal. The ... were because I was too lazy to copy the whole sentence!!!